src/internal/clusterstate/v2.8.0/clusterstate.go

```diff
@@ -0,0 +1,30 @@
+package v2_8_0
+
+import (
+	"context"
+
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+	"github.com/pachyderm/pachyderm/v2/src/internal/migrations"
+)
+
+func Migrate(state migrations.State) migrations.State {
+	return state.
+		Apply("Create pfs schema", func(ctx context.Context, env migrations.Env) error {
+			if err := createPFSSchema(ctx, env.Tx); err != nil {
+				return errors.Wrap(err, "creating pfs schema")
+			}
+			return nil
+		}).
+		Apply("Create pfs.repos table", func(ctx context.Context, env migrations.Env) error {
+			if err := createReposTable(ctx, env.Tx); err != nil {
+				return errors.Wrap(err, "creating pfs.repos table")
+			}
+			return nil
+		}).
```

Comment:
This should use the new `migrations.Squash` functionality so that it all happens at once.  Add a squash for every one except for the first one.  It's just like in git during a rebase.

Reply:
This will also be included in changes in the other branch